android/engine/src/test/java/org/smartregister/fhircore/engine/task/FhirCarePlanGeneratorTest.kt

```diff
@@ -723,6 +739,195 @@ class FhirCarePlanGeneratorTest : RobolectricTest() {
       }
   }
 
+  @Test
+  fun testGenerateCarePlanForANCVisitWithCql() = runTest {
```

Comment:
```suggestion
  fun testGenerateCarePlanForAncVisitWithCql() = runTest {
```

the convention is to not uppercase acronyms, like CQL